Source/Core/Core/PowerPC/Jit64/rangeset.h

```diff
@@ -0,0 +1,986 @@
+ /*
```

Comment:
This should likely be moved into externals. If the header is to remain within our main codebase, then it should be complying with our coding style (which it currently does not)